src/main/java/io/confluent/connect/jdbc/sink/dialect/SqlServerDialect.java

```diff
@@ -31,7 +31,7 @@
 public class SqlServerDialect extends DbDialect {
 
   public SqlServerDialect() {
-    super(getSqlTypeMap(), getLogicalToSqlTypeMap(), "[", "]");
+    super("[", "]");
   }
 
   private static Map<Schema.Type, String> getSqlTypeMap() {
```

Comment:
I think you forget to delete this map (unless you intentionally left it in for this dialect).